### ISSUE TITLE
Getting "Kubenet does not have netConfig. This is most likely due to lack of PodCIDR" as of today

### ISSUE THREAD

**Issue by Zimmergren** [Azure/AKS#273]
Hi,

Another issue that keeps appearing since yesterday/this morning is: 

```
network is not ready: [runtime network not ready: NetworkReady=false reason:NetworkPluginNotReady message:docker: network plugin is not ready: Kubenet does not have netConfig. This is most likely due to lack of PodCIDR]
```

This happens on one of my Daemon Sets; After deleting the daemon set and re-deploying it, I got the same issue again.

My other deployments work - and this has worked all up until yesterday. 

I have *not*:
- Done any upgrades or maintenance work on the cluster
- Changed any configurations in the daemonset, deployments or other resources in AKS and Kube
- Done anything else other than monitoring my cluster using Log Analytics

I have: 
- Scaled the cluster from N nodes to N+1 nodes
- The new node has a state of "Ready: Unknown", likely causing the above.

TLDR: 
- Do we have _more_ known issues with node scaling? 
- Every second/third time I scale my cluster, it breaks with random issues like this one. 
- Estimated about every ~10th time I do scaling, I have to delete the entire cluster and start over due to the handshake timeouts, kubectl and cluster unresponsiveness and general inaccessibility of the cluster after triggering a node scale.

**Comment by Zimmergren** [comment]
I've reproduced and verified that this issue happens when this other issue ([#274](https://github.com/Azure/AKS/issues/274)) is happening. Closing this.
